# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../embutils/parallelism.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2006-08-03 06:54:04.000000000 -0400
+++ ./Makefile	2007-03-28 00:21:50.000000000 -0400
@@ -33,13 +33,11 @@
 
 #CFLAGS=-pipe -g
 
-$(OBJDIR)/%.o: %.c
-	@test -d $(OBJDIR) || mkdir $(OBJDIR)
+$(OBJDIR)/%.o: %.c $(OBJDIR)
 	$(DIET) $(CROSS)gcc $(CFLAGS) $(FLAGS) -c $< -o $@
 	$(CROSS)strip -x -R .comment -R .note $@
 
 $(OBJDIR)/%.o: %.S $(OBJDIR)
-	@test -d $(OBJDIR) || mkdir $(OBJDIR)
 	$(DIET) $(CROSS)gcc -pipe -c $< -o $@
 
 $(OBJDIR)/%: $(OBJDIR)/%.o
@@ -51,13 +49,13 @@
 $(OBJDIR)/renice: $(OBJDIR)/renice.o $(OBJDIR)/fmt_long.o $(OBJDIR)/fmt_ulong.o
 
 $(OBJDIR)/cp: $(OBJDIR)/mv
-	ln -s mv $@
+	ln -sf mv $@
 
 $(OBJDIR)/chgrp: $(OBJDIR)/chown
-	-ln -s chown $@
+	ln -sf chown $@
 
 $(OBJDIR)/mkfifo: $(OBJDIR)/mknod
-	ln -s mknod $@
+	ln -sf mknod $@
 
 $(OBJDIR)/mknod.o $(OBJDIR)/mkdir.o: error.c parsemode.c
 
@@ -100,9 +98,9 @@
 install: $(TARGETS)
 	test -d $(DESTDIR)$(prefix)/bin || mkdir -p $(DESTDIR)$(prefix)/bin
 	$(INSTALL) $(TARGETS) $(DESTDIR)$(prefix)/bin
-	-ln -f $(DESTDIR)$(prefix)/bin/chown $(DESTDIR)$(prefix)/bin/chgrp
-	-ln -f $(DESTDIR)$(prefix)/bin/mv $(DESTDIR)$(prefix)/bin/cp
-	-ln -f $(DESTDIR)$(prefix)/bin/mknod $(DESTDIR)$(prefix)/bin/mkfifo
+	-ln -sf chown $(DESTDIR)$(prefix)/bin/chgrp
+	-ln -sf mv $(DESTDIR)$(prefix)/bin/cp
+	-ln -sf mknod $(DESTDIR)$(prefix)/bin/mkfifo
 
 uninstall:
 	rm -f $(patsubst %,$(DESTDIR)$(prefix)/bin/%,$(PRGS))
